### PR TITLE
fix: resolve symbol conflicts in wakelock_plus by updating Pigeon prefix

### PR DESCRIPTION
**Pull request by weitsai**
Updated the Pigeon configuration for wakelock_plus to use a unique ObjCOptions.prefix ("WKP") to prevent symbol conflicts with other plugins such as [image_picker_ios](https://github.com/flutter/packages/blob/main/packages/image_picker/image_picker_ios/pigeons/messages.dart#L14).

Changes include:
- Modified the Pigeon prefix from "FLT" to "WAKELOCKPLUS" in `messages.dart`.
- Regenerated the Pigeon files to reflect the new prefix.
- Cleaned and rebuilt the project to ensure compatibility.

### CODE REVIEW

File: wakelock_plus/pigeons/messages.dart [hunk 1]
```diff
@@ -16,7 +16,7 @@ class IsEnabledMessage {
   objcHeaderOut: 'ios/Classes/messages.g.h',
   objcSourceOut: 'ios/Classes/messages.g.m',
   objcOptions: ObjcOptions(
-    prefix: 'FLT',
+    prefix: 'WAKELOCK',
```
Comment (diegotori) [inline wakelock_plus/pigeons/messages.dart:19]: Change this to `WAKELOCKPLUS`.

**Review by diegotori — CHANGES_REQUESTED** [review]

File: wakelock_plus/pigeons/messages.dart (hunk 1 above)
Comment (weitsai) [reply wakelock_plus/pigeons/messages.dart]: Changed to `WAKELOCKPLUS`.

**Review by diegotori — APPROVED** [review]
LGTM

**Comment by diegotori** [comment]
@weitsai Thanks for your contribution. Once the automation goes green, then I'll release a new build.

**Comment by weitsai** [comment]
> @weitsai Thanks for your contribution. Once the automation goes green, then I'll release a new build.

Thanks for the update! I'll wait for the automation to complete. Let me know if there's anything else I can assist with.

**Comment by diegotori** [comment]
@weitsai I'm taking over your PR to the finish line, mainly to regenerate the Pigeons.

**Comment by weitsai** [comment]
Ok, thanks!

**Comment by diegotori** [comment]
@weitsai deployed as `wakelock_plus` and `wakelock_plus_platform_interface`, versions `1.2.10` and `1.2.2` respectively.

**Pull request merged**